src/openrct2-ui/windows/Scenery.cpp

```diff
@@ -836,14 +838,22 @@ static void WindowSceneryUpdate(rct_window* w)
     }
 }
 
+static void WindowSceneryContentScrollGetSize(rct_window* w, int32_t* height)
+{
+    auto rows = WindowSceneryCountRows(w);
+    *height = WindowSceneryRowsHeight(rows);
+}
+
```

Comment:
Why making it an out parameter and not a return value? Sounds more complex than it should be
```suggestion
static int32_t WindowSceneryContentScrollGetSize(rct_window* w)
{
    auto rows = WindowSceneryCountRows(w);
    return WindowSceneryRowsHeight(rows);
}

```

Reply:
that can be addressed when it is refactored into a class. As is for now it follows the style of all the other windows.

Reply:
Ok